cmake/find/FindGBenchmark.cmake

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2015-2016, University College London
```

Comment:
Instead of adding `FindGBenchmark.cmake` it's better to patch CMakeLists.txt so `GBenchmarkConfig.cmake` generated automatically. There are a few pitfalls here, for example this code doesn't work correctly with Release/Debug build types, custom build types. For Windows static library have '*.lib' suffix, not '*.a'. Also I'm not sure about `find_library` call. It probably can find system library instead of Hunter one. And components is not handled, this configuration should report error:
```cmake
find_package(GBenchmark REQUIRED COMPONENTS missing)
```